aws/kops-aws-platform/alb-ingress.tf

```diff
@@ -0,0 +1,42 @@
+variable "kops_alb_ingress_enabled" {
+  description = "Set to false to prevent the alb ingress from creating IAM resources"
+  default     = "false"
+}
+
+module "kops_alb_ingress" {
+  #source       = "git::https://github.com/cloudposse/terraform-aws-kops-alb-ingress.git?ref=tags/0.1.0"
+  source       = "../../../terraform-aws-kops-aws-alb-ingress"
```

Comment:
Use http path

Reply:
Waiting for `terraform-aws-kops-alb-ingress` repo to be made public.

Reply:
fixed in https://github.com/cloudposse/terraform-root-modules/pull/106/commits/945bd6e4f32f9b58bcbbe23730b4b8d251f381c1